Assets/Scripts/CameraMovement.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+
+class CameraMovement : ICameraMovementController {
+    // This is a reference to the transform of the camera.
+    private Camera camera_;
+
+    /*
+     * This sets the camera to the given transform.
+     */
+    public void SetCamera(Camera camera) {
+        camera_ = camera;
+    }
+
+
+    /*
+     * This handles the input for the camera and moves the camera with the appropriate vector.
+     * This is called every frame.
+     */
+    public void HandleInput() {
+        if (Input.GetKey(KeyCode.W)) {
```

Comment:
This doesn't follow the coding style, `if` statements do not get a space before the opening `(`.